apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -492,21 +492,21 @@ const DanceAiModal: React.FunctionComponent = () => {
 
   const headerValue = () => {
     return (
-      <div
+      <span
```

Comment:
Do we need to use `span` or could we just use `display: inline` in the right place?

Reply:
Similar question for the other places we changed `<div> ` to `<span>`...

Reply:
Isn't it more semantic HTML to use span for these? I didn't try `display: inline` (isn't that what span is for)? I was having some issues with the placeholder for the emojis showing up on a new line as I was working through this, possible I could rework though

Reply:
It [sounds](https://stackoverflow.com/a/5545914) like a `<span>` shouldn't wrap a block element, so that means we shouldn't use them to wrap things like the emojis.  From https://www.w3schools.com/tags/tag_span.asp's top-level summary, it feels like we should mostly use `<span>` to wrap text when we want to style it, though the spec is probably a bit more nuanced.

Reply:
Oh good call re: EmojiIcon. Lemme take another pass

Reply:
Looks fine with divs instead of spans I think, give it a look!